packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'terra-button';
+import IconLeft from 'terra-icon/lib/icon/IconLeft';
+import IconClose from 'terra-icon/lib/icon/IconClose';
+import ResponsiveElement from 'terra-responsive-element';
+import 'terra-base/lib/baseStyles';
+import ActionHeaderContainer from './_ActionHeaderContainer';
+
+const propTypes = {
+  /**
+   * Text to be displayed as the title in the header bar.
+   */
+  title: PropTypes.string,
+  /**
+   * Optionally sets the heading level. One of `1`, `2`, `3`, `4`, `5`, `6`. Default `level=1`.
+   */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
+  /**
+   * Callback function for when the close button is clicked.
+   * On small viewports a back button will be displayed instead of a close button when a separate onBack callback is not set.
```

Comment:
I think this needs cleanup as well. Instead of saying a back button will be displayed, can we instead say "On small viewports, this will be triggered by a back button if onBack is not set."

Reply:
Description updated [here](https://github.com/cerner/terra-core/commit/85b35f2f932e64ced0e37458cd33fde2f0633fe7#diff-adf8c495af771315ac46e269a49ed8bcR23).